### PR TITLE
Removed Redis queues code

### PR DESCRIPTION
**Pull request by KAbakumovAtl**
This is the first PR which completely removes Redis queues code.

In the next PR I am going to get rid of `bull` queues dependency. 

It is quite hard to do now, because it will require some minor refactoring of the `src/sync/installation.ts` 

I decide to do it in a separate PR so it will be easier to review

### CODE REVIEW

File: src/api/index.ts [hunk 1]
```diff
@@ -246,80 +242,16 @@ router.post(
...
-
-		// removing duplicate jobs
-		await Promise.all(duplicateJobs.map((job: Job) => {
-			logger.info({ job }, "removing duplicate job");
-			job.remove();
-		}));
-
-		res.send(`${duplicateJobs.length} duplicate jobs killed with fire.`);
+	async (_: Request, res: Response): Promise<void> => {
+		res.send(`This endpoint doesn't do anything useful anymore`);
```
Comment (bgvozdev) [inline src/api/index.ts:246]: Maybe we could just kill the whole endpoint?

**Review by bgvozdev — APPROVED** [review]

File: src/api/index.ts (hunk 1 above)
Comment (thombergs) [reply src/api/index.ts:246]: yes, kill the whole endpoint! It's not being called anymore, anyways.

File: src/api/index.ts [hunk 2]
```diff
@@ -246,80 +242,16 @@ router.post(
...
 	}
 );
 
 router.post(
 	"/requeue",
 	bodyParser,
-	async (request: Request, res: Response): Promise<void> => {
```
Comment (thombergs) [inline src/api/index.ts:289]: same here, kill the whole endpoint!

File: src/worker/startup.ts [hunk 3]
```diff
@@ -1,146 +1,17 @@
...
-// Start function for Node cluster worker
 export async function start() {
 	if (running) {
 		logger.debug("Worker instance already running, skipping.");
 		return;
 	}
 
 	logger.info("Micros Lifecycle: Starting queue processing");
-	// exposing queue metrics at a regular interval
-	timer = setInterval(sendQueueMetrics, 60000);
```
Comment (thombergs) [inline src/worker/startup.ts:130]: do we still have charts with these metrics? Or are we emitting the same metrics with the new queues?

**Review by thombergs — APPROVED** [review]

**Pull request merged**